pytest_reportportal/plugin.py

```diff
@@ -188,7 +192,10 @@ def pytest_configure(config):
     if not config.option.rp_launch_description:
         config.option.rp_launch_description = config.\
             getini('rp_launch_description')
-
+    if not config.option.rp_rerun:
+        config.option.rp_rerun = config.getini('rp_rerun')
+    if not config.option.rp_rerun_of:
+        config.option.rp_rerun_of = config.getini('rp_rerun_of')
```

Comment:
```suggestion
        config.option.rp_rerun_of = config.getoption('rp_rerun_of', default=None)
        config.option.rp_rerun = bool(config.option.rp_rerun_of)
```

Reply:
this won't work correctly - it is possible to use `rp_rerun` without `rp_rerun_of` (in which case the recent matching launch is used). your suggestion would disable such functionality.

Also, i don't get this:
```
config.option.rp_rerun_of = config.getoption('rp_rerun_of', default=None)
```
If i can't see the `rp_rerun_of` in inline options, i want to check the ini file,....i don't think your suggestion does that. I'd assume this was a typo, but `config.getini` [does not support "default"](https://docs.pytest.org/en/reorganize-docs/writing_plugins.html#_pytest.config.Config.getini)

Reply:
I believe this perfectly covers it all, including your above comment (implicit `rp_rerun = True` if `rp_rerun_of is not None`):

```
    if not config.option.rp_rerun_of:
        config.option.rp_rerun_of = config.getini('rp_rerun_of')
    if config.option.rp_rerun_of:
        config.option.rp_rerun = True
    else:
        if not config.option.rp_rerun:
            config.option.rp_rerun = config.getini('rp_rerun')
```

Reply:
> it is possible to use `rp_rerun` without `rp_rerun_of` (in which case the recent matching launch is used...

Where did you find this information? I'm not sure. It is missing in the documentation.

Reply:
https://github.com/reportportal/documentation/blob/master/src/md/src/DevGuides/rerun.md#latest-launch

Reply:
Ok, I have not seen that doc before...